types/office-js-preview/index.d.ts

```diff
@@ -8864,6 +8864,11 @@ declare namespace Office {
          * [Api set: Mailbox 1.1]
          *
          * **{@link https://docs.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose or Read
+         * 
+         * **Important**: A `recipientType` property value isn't returned by the 
+         * {@link https://docs.microsoft.com/en-us/javascript/api/outlook/office.from?view=outlook-js-preview | Office.context.mailbox.item.from.getAsync} 
+         * and {@link https://docs.microsoft.com/en-us/javascript/api/outlook/office.organizer?view=outlook-js-preview | Office.context.mailbox.item.organizer.getAsync} methods.
```

Comment:
Any reason why you can't link directly to getAsync (maybe to the 1st overload) on From and Organizer?